scripts/support/runner/arc.py

```diff
@@ -16,7 +16,7 @@
 
 
 class ArcBinaryRunner(ZephyrBinaryRunner):
-    '''Runner front-end for the ARC architecture, using openocd.'''
+    '''Runner front-end for the EM Starterkit board, using openocd.'''
```

Comment:
If that's true, why is this still named arc.py? Is this really not useful for any other boards?

Reply:
Not trying to stand in the way, just honestly confused about what this script's purpose is

Reply:
well, there is nothing arc specific about this, it is just "flashing" using gdb, i.e. it is not really flashing and just loading the ELF into memory, calling this a runner for the ARC architecture is wrong and probably the name should be changed and the script made to be more generic to handle this case on other boards where this is the only way to load an image on a device.